### DISCUSSION TITLE
Error while connecting to MySQL DB with Prisma (from Vercel)

### DISCUSSION THREAD

**Discussion by SoozaV** [vercel/community#1112] [Help] [answered]
Hi!

I'm trying to connect to a MySQL DB, I made a [deploy](https://notes-app-orpin-seven.vercel.app/) but I get this error when I try to create a new note:

_Invalid `prisma.note.create()` invocation: Authentication failed against database server at `162.241.61.208`, the provided database credentials for `numisver_admin` are not valid. Please make sure to provide valid database credentials for the database server at `162.241.61.208`._

The local db env variable is:

`DATABASE_URL="mysql://numisver_admin:paaF8DP4M,tX@162.241.61.208/numisver_num"`

and it works perfectlly fine when I run my app locally, I get the message "note created!", but from vercel's deploy I get that Invalid invocation error.

I set this same env variable on my project settings with the same name:

![image](https://user-images.githubusercontent.com/23610892/204127558-4eb54a4e-746b-467f-813f-b50cffea5688.png)

I've tried using the host ip with and without the port (:3306) but the result is the same, it works locally but not from Vercel.

Regards!

**Comment by SoozaV** [comment, marked as answer]
My bad, I had to [whitelist all ip's in my hosting settings.](https://vercel.com/guides/how-to-allowlist-deployment-ip-address#allow-all-ip-addresses).

Regards.

**Reply by amyegan** [reply]
Thanks for coming back and sharing your solution! 🙇
